docs/man_pages/publishing/appmanager-upload-android.md

```diff
@@ -13,9 +13,14 @@ WARNING: This command is not applicable to mobile website projects. To view the
 ### Options
 * `--certificate` - Sets the certificate that you want to use for code signing your Android app. You can set a certificate by index or name. <% if(isHtml) { %>To list available certificates, run `$ appbuilder certificate`<% } %> 
 * `--download` - If set, downloads the application package to the root of the project.
+* `--publish` - If set, the application package will be automatically published and visible for the members of its distribution groups. To add distribution groups, use `--groups` option.
+* `--send-email` - If set, an email with the link to the app will be send to all eligible users.
```

Comment:
We are currently checking the help files. I think it will be useful to warn the user in case he had not specified --publish, but he had specified --send-email or --send-push
For example in AppManagerService's upload method you can write something like:
```TypeScript
if(!options.publish && options.sendEmail) {
   this.$logger.warn("You have not specified --publish, so users will not receive email");
}

if(!options.publish && options.sendPush) {
   this.$logger.warn("You have not specified --publish, so users will not receive push notification.");
}
```

NOTE: The messages are just sample, you can change them as you think.